onnxruntime/python/tools/transformers/huggingface_models.py

```diff
@@ -89,11 +89,11 @@
     "albert-xlarge-v2": (["input_ids"], 12, True, "bert"),
     "albert-xxlarge-v2": (["input_ids"], 12, True, "bert"),
     # T5
-    #"t5-small": (["input_ids"], 11, False, "bert"),
-    #"t5-base": (["input_ids"], 11, False, "bert"),
-    #"t5-large": (["input_ids"], 11, False, "bert"),
-    #"t5-3b": (["input_ids"], 11, False, "bert"),
-    #"t5-11b": (["input_ids"], 11, False, "bert"),
+    "t5-small": (["input_ids"], 11, False, "bert"),
+    "t5-base": (["input_ids"], 11, False, "bert"),
+    "t5-large": (["input_ids"], 11, False, "bert"),
+    "t5-3b": (["input_ids"], 11, False, "bert"),
+    "t5-11b": (["input_ids"], 11, False, "bert"),
```

Comment:
It is really good that the tool can benchmark almost all the transformer based models